homeassistant/components/mill/climate.py

```diff
@@ -204,6 +204,19 @@ def __init__(self, coordinator):
         """Initialize the thermostat."""
         super().__init__(coordinator)
         self._attr_name = coordinator.mill_data_connection.name
+        if coordinator.mill_data_connection.mac_address:
+            self._attr_unique_id = coordinator.mill_data_connection.mac_address
+            self._attr_device_info = DeviceInfo(
+                identifiers={
```

Comment:
We should use `connections` instead of `identifiers` here

Reply:
Thanks for flagging. Can you elaborate on the difference? I'm reading the docs https://developers.home-assistant.io/docs/device_registry_index/ but it doesn't explain what it is other than repeating the words of the variable naems.

Reply:
When you use connections and there are multiple integrations that no about the device (ie unifi), they will only create on device entry.

`connections={(dr.CONNECTION_NETWORK_MAC, config.mac)}`